clarify interested parties (#147)
cncf/cnf-wg: interested-parties.md

# Interested Parties

> The interested parties listed below are used for voting purposes as outlined in the [Governance doc](GOVERNANCE.md). See the [contributing doc](CONTRIBUTING.md) for more information about getting involved and participating in the community.

Author: ```suggestion

The interested parties listed below are used for voting purposes as outlined in the [Governance doc](GOVERNANCE.md). See the [contributing doc](CONTRIBUTING.md) for more information about getting involved and participating in the community.
```

# Interested Parties

To express your interest in the cloud native networking topics under discussion in this working group, please add your name and organization name to this document via a Pull Request.

> Being listed in this document may grant you some rights within the group, such as voting capabilities as outlined in the [Governance doc](GOVERNANCE.md).  See the [contributing doc](CONTRIBUTING.md) for more information about getting involved and participating in the community. 

Reviewer: ```suggestion
Being listed in this document may grant you some rights within the group, such as voting capabilities as outlined in the [Governance doc](GOVERNANCE.md).  See the [contributing doc](CONTRIBUTING.md) for more information about getting involved and participating in the community.
```

Reviewer: remove trailing space per linter error

# Interested Parties

> The interested parties listed below are used for voting purposes as outlined in the [Governance doc](GOVERNANCE.md). See the [contributing doc](CONTRIBUTING.md) for more information about getting involved and participating in the community.

Reviewer 2: ```suggestion
To express your interest in the cloud native networking topics under discussion in this working group, please add your name and organization name to this document via a Pull Request.

Being listed in this document may grant you some rights within the group, such as voting capabilities as outlined in the [Governance doc](GOVERNANCE.md).  See the [contributing doc](CONTRIBUTING.md) for more information about getting involved and participating in the community. 

Note: adding yourself to this list does not imply any obligation (including legal) for you nor your organization.
```

Reviewer 2: @xmulligan this change would address #87 items including the concerns (via the note) while keeping it open for other uses.

Reviewer 3: I'm good with this suggestion.
